api-doc/openapi.yaml

```diff
@@ -228,42 +221,9 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/IncentiveCodeInput'
-      responses:
-        '201':
-          $ref: '#/components/responses/OkReturnCode'
-  /incentive_codes/{incentiveCode}:
-    parameters:
-      - name: incentiveCode
-        in: path
-        required: true
-        description: incentive code.
-        schema:
-          type: string
-    get:
-      description: Get incentive codes by code.
-      responses:
-        '200':
-          $ref: '#/components/responses/IncentiveCodesOk'
-    patch:
-      description: Patch incentive code
-      security:
-        - basicAuth: []
-      requestBody:
-        required: true
-        content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/IncentiveCodePatch'
       responses:
         '200':
-          $ref: '#/components/responses/Empty'
-    delete:
-      description: Delete incentive code
-      security:
-        - basicAuth: []
-      responses:
-        '204':
-          $ref: '#/components/responses/Empty'
+          $ref: '#/components/responses/OkReturnCode'
```

Comment:
Shouldn't this be 201 with the returned code in the response or am I misremembering how it's supposed to work?

Reply:
True, fixed